modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -158,278 +160,287 @@ Deploying myMuleApp ...
---
-Only running applications count toward workers usage. +
-`Stopped` applications do not consume vCore availability.
---
+. Sign in to Anypoint Platform.
+. Select *Runtime Manager* and change to your production environment.
+. In the *Applications* page, click *Deploy application*.
++
+You see the *Deploy Application* page:
++
+.The screenshot shows (*1*) the *Application Name* field, (*2*) the *Get from sandbox* button, and (*3*) the *Deploy Application* button on the *Deploy Application* page.
+image::cloudhub-deploy-app-sandbox.png[Application Name field and Get from sandbox and Deploy Application buttons]
+. Provide a name for your application and click *Get from sandbox*:
++
+This button is available only if a non-production environment exists.
+. Select your sandbox environment and select the application to move to production.
++
+Only applications that are deployed to CloudHub appear in the list.
+You can promote only applications deployed to CloudHub to run on other CloudHub workers.
+. If you want to copy the environment variables and Mule version from the source application, select *Merge environment variables and mule version*.
+. Click *Apply*.
+. Click tabs to configure application options.
++
+See <<configure-application-settings,Configure Application Settings>>.
+. Click *Deploy Application*.
++
+See <<Deployment Progress>>.
 
-Worker sizes have different compute and memory capacities, including:
+== Application Names 
 
-[%header,cols="2*a"]
-|===
-|Worker Size |Heap Memory
-|0.1 vCores |500 MB
-|0.2 vCores |1 GB
-|1 vCore |1.5 GB
-|2 vCores |3.5 GB
-|4 vCores |7.5 GB
-|8 vCores |15 GB
-|16 vCores |32 GB
-|===
+The application name identifies your application in Runtime Manager and also used for the 
+public application name in the `cloudhub.io` domain.
+For example, an application named `myapplication` is accessible at `+http://myapplication.cloudhub.io+`.
 
-Workers that have less than 1 vCore capacity (0.1 vCores and 0.2 vCores) offer limited CPU and IO for smaller work loads. Each worker has 8 GB of storage, which is used for both system and application storage.
+The application name must be globally unique across CloudHub to avoid domain conflicts.
+Runtime Manager prevents you from choosing a name that is already in use.
 
-[NOTE]
---
-Workers running on fractional vCores (0.1 vCores and 0.2 vCores) have the ability to burst to higher CPU speeds for a short duration of time. This is useful for improving application start-up times and with processing infrequently large workloads. Allocating workers with 1 vCore or greater is recommended if consistency of performance is needed.
---
+To avoid domain conflicts, create a naming convention for your applications, such as adding a prefix to 
+your applications: `mycompany-mydept-myapplication`.
+You can then add DNS records to hide this complex application name.
+For example, you can route requests to `myapplication.mycompany.com` to `mycompany-mydept-myapplication.cloudhub.io`.
```

Comment:
This para seems a bit choppy to me. 
It repeats the thing about avoiding domain conflicts.
It makes its unique point about having a naming convention that enables you to group apps but also make them distinct. Good. Although that prefix is longer than the root, which is not something I typically expect of a prefix.
Then it makes another unique point about hiding the complex name by using DNS records. Good.
But the example does not make sense to me: how does that "simple" name route to the complex one, when the elements are not in the same order in each?

Reply:
Re: example - the point is that the DNS record maps the hierarchical name to something that people typically access apps at (blah.company.com).

I just tested - RTM doesn't ensure unique names!
Clarified.